ovos_PHAL_plugin_homeassistant/__init__.py

```diff
@@ -121,6 +122,14 @@ def get_brightness_increment(self) -> int:
         """
         return self.config.get("brightness_increment", 10)
 
+    def get_search_confidence_threshold(self) -> int:
```

Comment:
This could be simplified to:
```python
@property
def search_confidence_threshold(self):
```
Then config changes are handled after init without having to restart PHAL/enclosure service